source/docs/06-neumes.xml

```diff
@@ -147,19 +147,19 @@
                         <cell>
                            <figure>
                               <head>nc-1</head>
-                              <graphic url="../images/v4/modules/neumes/NC-rhombusNEW.png"/>
+                              <graphic url="../v4/images/modules/neumes/NC-rhombusNEW.png"/>
```

Comment:
@kepper @bwbohl @ahankinson Is there a reason why these paths are relative whereas all other paths are targetting the guidelines directory (repo?) itself?

Reply:
Also, in other lines (e.g. 185), the graphic urls are referring to a folder in the main repo:

`<graphic url="../images/modules/neumes/salicus.png"/>`

Reply:
If everything else is doing it one way, it's probably better to change these to match.

Reply:
Targetting the guidelines repository looks wrong. Do you have an example?

Reply:
See e.g line [128](https://github.com/music-encoding/music-encoding/blob/106bbdeacd25d5bb2d1af24ffa47853ca5b1b5d8/source/docs/06-neumes.xml#L128), or all other lines starting with `/guidelines/...`. (If I see it correctly, there is no guidelines folder in this repo any more)

There are some (but not all) of the images relevant to the chapter here: https://github.com/music-encoding/music-encoding/tree/develop/source/images/modules/neumes, but much more images over there in the guidelines repo: https://github.com/music-encoding/guidelines/tree/master/v4/images/modules/neumes

Reply:
Thanks, @musicEnfanthen! So it would be good to clean this up and make sure `v4` and `dev`. Changing this in the source will not update/fix the Guidelines for MEI4, if I'm not mistaken. So these specific changes have to be made [here](https://github.com/music-encoding/guidelines/blob/master/v4/content/neumes.html).

Reply:
@rettinghaus, I thought we shouldn't edit the guidelines repository at all, and that all changes have to be made here in this music-encoding repository. From the README of the guidelines repo:
>The content of this repository should not be edited directly. Contributions to the MEI Guidelines should be made directly to the music-encoding source repository.

I guess to fix both versions of the guidelines (MEI4 and dev), the changes should be done in both the `stable` branch and the `develop` branch of this repo?

Reply:
@martha-thomae fixing the music-encoding repo (source) will only update the dev version of the guidelines. For fixing MEI4, we do need to edit the guidelines repos. But only in that case.

Reply:
Oh, got it. Thanks for the clarification, @lpugin!

Reply:
Thank you all for looking into this! We ([Yinan](https://github.com/yinanazhou) and I) changed the addresses and [pulled a request in the guideline repo](https://github.com/music-encoding/guidelines/pull/199). Please help us to check it. Thank you!